modules/images-imagestream-external-image-tags.adoc

```diff
@@ -10,9 +10,9 @@ You can add tags for external images.
 
 * Add tags pointing to internal or external images, by using the `oc tag` command
  for all tag-related operations:
- +
++
```

Comment:
There are other three '+' strings in this doc, need removed

Reply:
There was a spacing issue previously that caused the '+'s to build. I removed the space and it looks like it's building correctly to align the example context with the procedure.